docs/source/remote-data-services.rst

```diff
@@ -0,0 +1,155 @@
+Remote Data Services
+====================
+
+As described in Section :doc:`Internal Data Ingestion <bytes>`, various
+user-facing functions such as ``dataframe.read_parquet`` and lower level
```

Comment:
Might include `dd.read_csv` here alongside `dd.read_parquet`.  I suspect that read_csv is more broadly recognized (for now!)